doc/developer/building-frr-for-fedora.rst

```diff
@@ -110,13 +110,12 @@ And load the kernel modules on the running system:
       sudo systemctl stop firewalld.service
       sudo iptables -F
 
-Install service files
-^^^^^^^^^^^^^^^^^^^^^
+Install frr Service
+^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: console
 
-   sudo install -p -m 644 redhat/frr.service /usr/lib/systemd/system/frr.service
-   sudo install -p -m 755 redhat/frr.init /usr/lib/frr/frr
```

Comment:
I think this needs to stay, and add `frrinit.sh` too, right?

Reply:
frrinit.sh is handled by `make install`:
```
libtool: install: /usr/bin/install -c vrrpd/.libs/vrrpd /usr/lib/frr/vrrpd
 /usr/bin/mkdir -p '/usr/lib/frr'
 /usr/bin/install -c tools/frr-reload tools/frr-reload.py tools/frr tools/frrcommon.sh tools/frrinit.sh tools/generate_support_bundle.py tools/watchfrr.sh '/usr/lib/frr'
/usr/bin/mkdir -p "/usr/share/info"
```